docs/src/site/twiki/EntitySLAAlerting.twiki

```diff
@@ -0,0 +1,57 @@
+---++Entity SLA Alerting
+
+Falcon supports sla in feed and process.
+
+Types of sla supported for feed:
+
+   1.slaLow
```

Comment:
Somewhere (here or in EntitySpecification) you need to be more specific about what the SLA measures - what are the staring time and ending time that determine the time interval that is compare to the SLA values.